multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/security/SpaceGuidBasedAuthorizationFilter.java

```diff
@@ -3,36 +3,53 @@
 import java.io.IOException;
 import java.text.MessageFormat;
 
+import javax.inject.Inject;
+import javax.inject.Named;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import org.cloudfoundry.multiapps.controller.core.auditlogging.LoginAttemptAuditLog;
 import org.cloudfoundry.multiapps.controller.web.Messages;
 import org.cloudfoundry.multiapps.controller.web.util.SecurityContextUtil;
 import org.cloudfoundry.multiapps.controller.web.util.ServletUtil;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.web.server.ResponseStatusException;
 
+@Named
 public abstract class SpaceGuidBasedAuthorizationFilter implements UriAuthorizationFilter {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SpaceGuidBasedAuthorizationFilter.class);
 
     private final AuthorizationChecker authorizationChecker;
 
+    @Inject
+    private LoginAttemptAuditLog loginAttemptAuditLog;
```

Comment:
This is not right. The class is abstract and can't be mark as bean. The proper way is to add `LoginAttemptAuditlog` in the constructor and all child classes of `SpaceGuidBasedAuthorizationFilter` must inject `LoginAttemptAuditlog` bean.